pkgs/development/python-modules/cryptography/vectors.nix

```diff
@@ -1,12 +1,13 @@
 { buildPythonPackage, fetchPypi, lib, cryptography }:
 
 buildPythonPackage rec {
-  pname = "cryptography_vectors";
-  # The test vectors must have the same version as the cryptography package:
+  pname = "cryptography-vectors";
+  # The test vectors must have the same version as the cryptography package
   version = cryptography.version;
 
-  src = fetchPypi {
-    inherit pname version;
+  src = fetchPypi rec {
+    pname = "cryptography_vectors";
```

Comment:
Is that intended? Seems like the PyPI package also uses a dash: https://pypi.org/project/cryptography-vectors/

Reply:
Yes, the download links use `_` (see https://pypi.org/project/cryptography-vectors/#files) which makes no sense.

Reply:
Oh, my bad.